src/app/shared/example-viewer/example-viewer.ts

```diff
@@ -13,21 +16,26 @@ const fileExtensionRegex = /(.*)\.(\w+)/;
   styleUrls: ['./example-viewer.scss']
 })
 export class ExampleViewer {
+  @ViewChild('snippet') snippet: CodeSnippet;
```

Comment:
you should be able to just do `@ViewChild(CodeSnippet)` - but actually you might want to do `@ViewChildren(CodeSnippet)` because I don't think `ViewChild` updates